.github/workflows/backfill-ci.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
       fail-fast: false
       matrix:
         jdk: ['11'] # TODO: Enable java 17 tests https://issues.apache.org/jira/browse/CASSANDRA-16895
-        pulsarImageTag: ['2.10_0.4']
+        pulsarImage: ['datastax/lunastreaming:2.10_2.4', 'apachepulsar/pulsar:2.10.3', 'apachepulsar/pulsar:2.11.0']
```

Comment:
2.10_2.4 is very old
let's use 2.10_3.4

Reply:
Fixed. Also verified all tests are green>

@eolivelli PTAL